com.unity.multiplayer.mlapi/Runtime/Metrics/NetworkMetrics.cs

```diff
@@ -7,28 +9,40 @@ namespace MLAPI.Metrics
 {
     public interface INetworkMetrics
     {
-        void TrackNamedMessageSent(string messageName, ulong bytesCount);
+        void TrackNetworkObject(NetworkObject networkObject);
 
-        void TrackNamedMessageReceived(string messageName, ulong bytesCount);
+        void TrackNamedMessageSent(ulong clientId, string messageName, ulong bytesCount);
+
+        void TrackNamedMessageSent(List<ulong> clientIds, string messageName, ulong bytesCount);
```

Comment:
Oh man, I wish we had c# 8 because this could just be a default interface implementation